node-commons/src/main/java/org/apache/iotdb/commons/concurrent/ThreadName.java

```diff
@@ -84,4 +275,106 @@ public enum ThreadName {
+    for (ThreadName threadName : queryThreadNames) {
+      if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.QUERY;
+      }
+    }
+    if (givenThreadName.contains(MPP_COORDINATOR_SCHEDULED_EXECUTOR.getName())) {
+      return DataNodeThreadModule.MPP_SCHEDULE;
+    }
+    for (ThreadName threadName : compactionThreadNames) {
+      if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.COMPACTION;
+      }
+    }
+    for (ThreadName threadName : walThreadNames) {
+      if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.WAL;
+      }
+    }
+    if (givenThreadName.contains(MPP_COORDINATOR_WRITE_EXECUTOR.getName())) {
+      return DataNodeThreadModule.MPP_WRITE;
+    }
+    for (ThreadName threadName : flushThreadNames) {
+      if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.FLUSH;
+      }
+    }
+    for (ThreadName threadName : schemaEngineThreadNames) {
+      if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.SCHEMA_ENGINE;
+      }
+    }
+    for (ThreadName threadName : clientServiceThreadNames) {
+      if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.CLIENT_SERVICE;
+      }
+    }
+    for (ThreadName threadName : iotConsensusThrreadNames) {
+      if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.IOT_CONSENSUS;
+      }
+    }
+    for (ThreadName threadName : ratisThreadNames) {
+      if (threadName.getName().contains("\\d")) {
+        if (Pattern.compile(threadName.getName()).matcher(givenThreadName).find()) {
+          return DataNodeThreadModule.RATIS_CONSENSUS;
+        }
+      } else if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.RATIS_CONSENSUS;
+      }
+    }
+    for (ThreadName threadName : computeThreadNames) {
+      if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.COMPUTE;
+      }
+    }
+    for (ThreadName threadName : syncThreadNames) {
+      if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.SYNC;
+      }
+    }
+    for (ThreadName threadName : jvmThreadNames) {
+      if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.JVM;
+      }
+    }
+    if (givenThreadName.contains(LOG_BACK.getName())) {
+      return DataNodeThreadModule.LOG_BACK;
+    }
+    for (ThreadName threadName : metricsThreadNames) {
+      if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.METRICS;
+      }
+    }
+    for (ThreadName threadName : otherThreadNames) {
+      if (givenThreadName.contains(threadName.getName())) {
+        return DataNodeThreadModule.OTHER;
+      }
+    }
+    log.error("Unknown thread name {}", givenThreadName);
```

Comment:
Using debug log and printing the log only means that the metric module has encountered a thread that is not known in advance when counting threads, and the issue does not affect the system's operation